common/djangoapps/student/models.py

```diff
@@ -288,6 +288,8 @@ class Meta(object):
         blank=True, null=True, max_length=6, db_index=True,
         choices=LEVEL_OF_EDUCATION_CHOICES
     )
+    profession = models.TextField(blank=True, null=True)
+    specialty = models.TextField(blank=True, null=True)
```

Comment:
The company, title, first_name, and last_name extra fields are good examples to use for figuring out how to persist to UserProfile.meta.